pkg/sql/parsers/dialect/mysql/scanner_test.go

```diff
@@ -21,6 +21,15 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/sql/parsers/dialect"
 )
 
+func TestScan(t *testing.T) {
```

Comment:
test for what

Reply:
sorry, I forget to delete it, I use it to test the sql parser result.